Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -762,8 +762,7 @@ class BrowserViewController: UIViewController {
     private func updateInContentHomePanel(url: NSURL?) {
         if !urlBar.inOverlayMode {
             if AboutUtils.isAboutHomeURL(url) {
-                let showInline = AppConstants.MOZ_MENU || ((tabManager.selectedTab?.canGoForward ?? false || tabManager.selectedTab?.canGoBack ?? false))
```

Comment:
I don't think so since this condition would have always been true if MOZ_MENU was set.